core/lib/snapshots_applier/src/lib.rs

```diff
@@ -261,6 +261,41 @@ impl SnapshotsApplierTask {
         }
     }
 
+    /// Checks whether the snapshot recovery is already completed.
+    ///
+    /// Returns `None` if no snapshot recovery information is detected in the DB.
+    /// Returns `Some(true)` if the recovery is completed.
+    /// Returns `Some(false)` if the recovery is not completed.
```

Comment:
Bikeshedding: IMO, it could make sense to introduce a dedicated enum with 3 variants instead here.

Reply:
I thought about it too, but IMHO this is not ambiguous and given that the method has very limited usage, it's OK to leave it as is.

Reply:
OTOH no strong opinion, so let's add it, why not.